src/cephadm/cephadm

```diff
@@ -2834,15 +2833,27 @@ def _get_parser():
 
     return parser
 
+def _parse_args(av):
+    parser = _get_parser()
+    args = parser.parse_args(av)
+
+    if not args.image:
+        if 'name' in args:
+            (type_, _) = args.name.split('.', 1)
```

Comment:
Will cause `ValueError`  in the case if the `crash` container ...

```suggestion
            if '.' in args.name:
                (type_, _) = args.name.split('.', 1)
            else:
                # assume name is the type (e.g. `crash`)
                type_ = args.name
```

Reply:
are you sure?
```
autriche:~ (master) 04:48 PM $ python2
Python 2.7.17 (default, Nov  7 2019, 10:07:09) 
[GCC 7.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> 'crash'.split('.', 1)
['crash']
>>> 
autriche:~ (master) 04:48 PM $ python3
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 'crash'.split('.', 1)
['crash']
>>> 
```

Reply:
oh, i should [0] not do the _ thing